tools/xmlcreate.py

```diff
@@ -196,6 +199,7 @@ def fix_spec(src, spec):
 parser = argparse.ArgumentParser(description="Generate an xml object file from a source file")
 parser.add_argument('file', help="overlay file to generate xml from")
 parser.add_argument('name', help='name to use for xml variables')
+parser.add_argument('segment_number', help='the segment number of the object')
```

Comment:
This should be an optional argument which defaults to 6.

Reply:
This file was something I changed a while back but didnt end up keeping but forgot to change back to the one in the repo.